### PR TITLE
work on tests

### PR DESCRIPTION
**Pull request by kyranb**

### CODE REVIEW

**Comment by smlparry** [comment]
3 more!

File: spec/views/welcome/index.html.erb_spec.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require 'rails_helper'
 
+#TODO Fix this test
```
Comment (smlparry) [inline spec/views/welcome/index.html.erb_spec.rb]: `# TODO: Fix this test` 

Whitespace is important to a sensitive ruby dev like me

Comment (smlparry) [inline spec/views/welcome/index.html.erb_spec.rb]: Also shouldn't be `Fix this test`, should be `Implement this test` as the test is just a pending one

**Comment by smlparry** [comment]
I dont see a green build? Were you meant to merge this?
Why is there no reply to my comment?

0/10 would not float at envato

**Comment by kyranb** [comment]
<?php

use Laravel;

//Busy generating cash

public function tests(){
Return null;

}

**Comment by smlparry** [comment]
ew what is that whitespace

**Pull request merged**